### ISSUE TITLE
Chromecast exception

### ISSUE THREAD

**Issue by emabartolomucci** [deanoemcke/thegreatsuspender#378]
I don't know if it's possible, but when casting a tab on Chromecast, the extension should be aware of this and not suspending that tab, as it happened to me yesterday while casting a sport event on a TV.

**Comment by janpio** [comment]
It's funny how e.g. Soundcloud keeps playing the music and even jumps to the next title in the playlist when the tab is suspended, but looses all context that it was playing something, what song it was playing and that I was casting this to my Chromecast Audio when I resume the tab. I wonder how casting is implemented to get this result...

**Comment by deanoemcke** [comment]
@janpio yeah, i've noticed with spotify on my mobile, it seems to be able to continue casting even if i force quit the app.
